### PR TITLE
[Localization] - apply localization for Personal Cabinet (Parent/Provider view) #1867

### PR DESCRIPTION
**Pull request by khrMykhailo**
Localization has been provided for Nav, and for two tabs in personal information

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -1,44 +1,98 @@
...
+    "TUESDAY": "Tue",
+    "WEDNESDAY": "Wed"
   },
   "DIRECTIONS": "Categories",
+  "DIRECTOR_NAME": "Director's full name",
   "DISPLAY_ON_THE_PAGE": "Items per page:",
+  "DISTRICT_OF_THE_CITY": "District of the city",
   "DO_YOU_PROVIDE_EDUCATIONAL_SERVICES": "Do you provide educational services?",
+  "EDIT": "Edit",
+  "EDPNOU": "EDPNOU",
```
Comment (litvinets) [inline src/assets/i18n/en.json:43]: EDRPOIPN : Unified State Register Code of Enterprises and Organizations of Ukraine

File: src/assets/i18n/en.json [hunk 2]
```diff
@@ -1,44 +1,98 @@
...
-    "SUNDAY": "Sun"
+    "SUNDAY": "Sun",
+    "THURSDAY": "Thu",
+    "TUESDAY": "Tue",
+    "WEDNESDAY": "Wed"
   },
   "DIRECTIONS": "Categories",
+  "DIRECTOR_NAME": "Director's full name",
   "DISPLAY_ON_THE_PAGE": "Items per page:",
+  "DISTRICT_OF_THE_CITY": "District of the city",
```
Comment (litvinets) [inline src/assets/i18n/en.json:40]: DISTRICT: City district

File: src/assets/i18n/en.json [hunk 3]
```diff
@@ -91,116 +153,123 @@
...
-      "INVALID_LENGTH_START": "This field must contain",
-      "INVALID_LENGTH_END": "symbols.",
       "REQUIRED_INPUT": "This field is required."
     }
   },
   "FOUND": "Found",
   "FOUNDER": "Founder",
   "FREE_WORKSHOP": "FREE WORKSHOP",
   "FROM": "From",
+  "FULL_ORGANIZATION_NAME": "Full Organization Name",
```
Comment (litvinets) [inline src/assets/i18n/en.json:221]: ORGANIZATION_FULL_NAME: Organization full name

File: src/assets/i18n/en.json [hunk 4]
```diff
@@ -91,116 +153,123 @@
...
   "INFO_MENU": {
     "UNLIMITED": "By selecting the \"Unlimited\" option, the group has unlimited of the number of users. If you enter the number of available places for users - upon reaching the limit, recruitment will be automatically closed and application submission will be stopped automatically."
   },
-  "RATE": "RATE",
+  "INFORMATION_ABOUT": "Information about",
+  "ITN": "ITN",
```
Comment (litvinets) [inline src/assets/i18n/en.json:226]: ITN - > IPN
individual tax number

File: src/assets/i18n/en.json [hunk 5]
```diff
@@ -91,116 +153,123 @@
...
-  "SEND_A_MESSAGE" : "Send a message",
-  "RECRUITMENT": "Recruitment",
-  "SEARCH_BY_DIRECTIONS": "Search by category",
-  "SELECT_AGE": "Select age",
-  "SHOW_ON_THE_MAP": "Show on the map",
-  "SIGN_UP": "SIGN UP",
-  "SIGN_UP_WE_WILL_HELP_YOU_ATTRACT_NEW_STUDENTS": "Sign up - we will help you attract new students!",
-  "APPLY_AN_APPLICATION": "APPLY FOR THE WORKSHOP",
-  "SUPPORT": "SUPPORT",
+  "TYPE_OF_INSTITUTION": "Type of institution",
```
Comment (litvinets) [inline src/assets/i18n/en.json:271]: INSTITUTION_ TYPE: Institution Type

File: src/assets/i18n/en.json [hunk 6]
```diff
@@ -91,116 +153,123 @@
...
+  "TYPE_OF_ORGANIZATION": "Organization type",
```
Comment (litvinets) [inline src/assets/i18n/en.json:272]: "ORGANIZATION_TYPE": "Organization type",

File: src/assets/i18n/en.json [hunk 7]
```diff
@@ -1,44 +1,98 @@
...
       "CHILD": "child",
       "CHILD_GENITIVE": "children",
+      "CHILDREN": "children",
       "CHILDREN_ABLATIVE": "children",
-      "NO_CHILDREN": "No children",
-      "CHILDREN": "children"
+      "NO_CHILDREN": "No children"
+    },
+    "NAV_BAR_NAME": {
+      "ABOUT_PORTAL": "About portal",
```
Comment (litvinets) [inline src/assets/i18n/en.json:54]: About the portal

File: src/assets/i18n/en.json [hunk 8]
```diff
@@ -1,44 +1,98 @@
...
+      "ADD_DIRECTION": "Add direction",
+      "ADD_INFORMATION_ABOUT_CHILD": "Add information about child",
+      "ADD_WORKSHOP_ADMIN": "Add workshop admin",
+      "ADMINISTRATION": "Administration",
+      "ADMINS": "Admins",
+      "APPLICATIONS": "Applications",
+      "CHAT": "Chat",
+      "DIRECTIONS": "Directions",
+      "EDIT_ACHIEVEMENT": "Edit achievement",
+      "EDIT_ADMIN": "Edit admin",
```
Comment (litvinets) [inline src/assets/i18n/en.json:67]: "EDIT_ADMIN": "Edit administrator",

File: src/assets/i18n/en.json [hunk 9]
```diff
@@ -1,44 +1,98 @@
...
+      "EDIT_DIRECTION": "Edit direction",
+      "EDIT_INFORMATION_ABOUT_CHILD": "Edit information about child",
```
Comment (litvinets) [inline src/assets/i18n/en.json:69]: duplication

File: src/assets/i18n/en.json [hunk 10]
```diff
@@ -1,44 +1,98 @@
...
+      "EDIT_INSTITUTION": "Edit institution",
```
Comment (litvinets) [inline src/assets/i18n/en.json:70]: "EDIT_PROVIDER": "Edit provider indormation",

File: src/assets/i18n/en.json [hunk 11]
```diff
@@ -1,44 +1,98 @@
...
+      "EDIT_PERSONAL_INFORMATION": "Edit personal information",
+      "EDIT_WORKSHOP": "Edit workshop",
+      "FAVORITE": "Favorite",
+      "HISTORY_OF_CHANGES": "History of changes",
+      "INFORMATION_ABOUT_CHILD": "Information about child",
```
Comment (litvinets) [inline src/assets/i18n/en.json:75]: I think this si redundant and can be removed
use already existed INFORMATION_ABOUT_CHILDREN

File: src/assets/i18n/en.json [hunk 12]
```diff
@@ -1,44 +1,98 @@
...
+      "INFORMATION_ABOUT_CHILDREN": "Information about children",
+      "INFORMATION_ABOUT_PROVIDER": "Information about provider",
```
Comment (litvinets) [inline src/assets/i18n/en.json:77]: "INFORMATION_ABOUT_PROVIDER": "Information about the provider",

File: src/assets/i18n/en.json [hunk 13]
```diff
@@ -1,44 +1,98 @@
...
+      "MAIN": "Main",
+      "MAIN_PAGE": "Main page",
```
Comment (litvinets) [inline src/assets/i18n/en.json:79]: duplication, I think we can use only one -> Main Page

File: src/assets/i18n/en.json [hunk 14]
```diff
@@ -1,44 +1,98 @@
...
+      "MESSAGES": "Messages",
+      "NEW_WORKSHOP": "New workshop",
```
Comment (litvinets) [inline src/assets/i18n/en.json:81]: NEW_WORKSHOP -> I think this is a wrong translation
for navbar, it should Add new workshop

File: src/assets/i18n/en.json [hunk 15]
```diff
@@ -1,44 +1,98 @@
...
+      "NORMATIVE_LEGAL_ACTS": "Normative legal acts",
```
Comment (litvinets) [inline src/assets/i18n/en.json:82]: remove this one and apply LAWS_AND_REGULATIONS

File: src/assets/i18n/en.json [hunk 16]
```diff
@@ -1,44 +1,98 @@
...
+      "PERSONAL_INFORMATION": "Personal information",
+      "PLATFORM": "Platform",
+      "PORTAL": "Portal",
+      "PROVIDERS": "Providers",
+      "REQUEST_ON_WORKSHOP": "Request on workshop",
```
Comment (litvinets) [inline src/assets/i18n/en.json:87]: New workshop application

File: src/assets/i18n/en.json [hunk 17]
```diff
@@ -1,44 +1,98 @@
...
+      "SEARCH_RESULT": "Search result",
+      "SUPPORT": "Support",
+      "TOP_DESTINATION": "Top destination",
```
Comment (litvinets) [inline src/assets/i18n/en.json:90]: "TOP_DESTINATION": "Top destination",
 remove and use TOP DIRECTIONS

File: src/assets/i18n/en.json [hunk 18]
```diff
@@ -1,44 +1,98 @@
...
+      "UPDATE_ADMIN": "Update admin",
```
Comment (litvinets) [inline src/assets/i18n/en.json:91]: do not use shorts, for translation it has to be the whole word -> administrator
don't use "update", stick with the one translation variant, in our case  EDIT

File: src/assets/i18n/en.json [hunk 19]
```diff
@@ -1,44 +1,98 @@
...
+      "UPDATE_DEPUTY_DIRECTOR": "Update deputy director",
+      "USER_INFO": "User info",
```
Comment (litvinets) [inline src/assets/i18n/en.json:93]: **do not use abbreviation for translation**
User information

File: src/assets/i18n/en.json [hunk 20]
```diff
@@ -1,44 +1,98 @@
...
+      "PERSONAL_INFORMATION": "Personal information",
+      "PLATFORM": "Platform",
+      "PORTAL": "Portal",
+      "PROVIDERS": "Providers",
+      "REQUEST_ON_WORKSHOP": "Request on workshop",
+      "SEARCH_RESULT": "Search result",
+      "SUPPORT": "Support",
+      "TOP_DESTINATION": "Top destination",
+      "UPDATE_ADMIN": "Update admin",
+      "UPDATE_DEPUTY_DIRECTOR": "Update deputy director",
```
Comment (litvinets) [inline src/assets/i18n/en.json:92]: "UPDATE_DEPUTY_DIRECTOR": "Update deputy director",
EDIT PROVIDER DEPUTY -> Edit provider deputy

File: src/assets/i18n/en.json [hunk 21]
```diff
@@ -1,44 +1,98 @@
 {
+  "ABBREVIATED_NAME_OF_THE_ORGANIZATION": "Abbreviated name of the organization",
   "ABOUT_THE_PORTAL": "ABOUT THE PORTAL",
+  "ACTUAL_ADDRESS": "Actual address",
   "ADD": "Add",
+  "ADD_ACHIEVEMENTS": "Add Achievements",
```
Comment (litvinets) [inline src/assets/i18n/en.json:6]: "ADD_ACHIEVEMENTS": "Add Achievements",
```suggestion
  "ADD_ACHIEVEMENT": "Add Achievement",
```

File: src/assets/i18n/en.json [hunk 22]
```diff
@@ -1,44 +1,98 @@
 {
+  "ABBREVIATED_NAME_OF_THE_ORGANIZATION": "Abbreviated name of the organization",
```
Comment (litvinets) [inline src/assets/i18n/en.json:2]: ABBREVIATED_NAME_OF_THE_ORGANIZATION - bad naming

  "FULL_ORGANIZATION_NAME": "Повна назва організації",
  "SHORT_ORGANIZATION_NAME": "Скорочена назва організації",

File: src/app/shell/personal-cabinet/shared-cabinet/user-config/user-config.component.html [hunk 23]
```diff
@@ -1,28 +1,28 @@
...
   </div>
   <div class="config">
-    <h4>Електронна пошта</h4>
+    <h4>{{ 'FORMS.LABELS.EMAIL' | translate }}</h4>
     <p style="margin-bottom: 0">{{ user.email }}</p>
-    <p class="link" (click)="onRedirect('/Account/ChangeEmail')">Змінити електронну пошту</p>
-    <h4>Пароль</h4>
-    <p class="link" (click)="onRedirect('/Account/ChangePassword')">Змінити Пароль</p>
+    <p class="link" (click)="onRedirect('/Account/ChangeEmail')">{{ 'CHANGE_EMAIL' | translate }}</p>
+    <h4>{{ 'PASSWORD' | translate }}</h4>
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/shared-cabinet/user-config/user-config.component.html:25]: please, teh kepp general tarnslation approach
change, update replace with edit

**Review by litvinets — CHANGES_REQUESTED** [review]

File: src/assets/i18n/en.json (hunk 9 above)
Comment (khrMykhailo) [reply src/assets/i18n/en.json:69]: No, it was ADD_INF, not EDIT

File: src/assets/i18n/en.json [hunk 24]
```diff
@@ -91,116 +148,124 @@
...
       "AVAILABLE_PLACES": "Number of available places",
-      "BIRTHDAY":  "Birthday",
+      "BIRTHDAY": "Birthday",
       "BUILDING": "Building",
       "DAYS_HOURS": "Days/Hours",
       "EMAIL": "Email",
       "KEYWORDS": "Keywords",
+      "LAST_NAME": "Last name",
       "MIDDLENAME": "Middle name",
       "NAME": "Name",
```
Comment (litvinets) [inline src/assets/i18n/en.json:174]: First name

File: src/assets/i18n/en.json [hunk 25]
```diff
@@ -1,44 +1,93 @@
...
   "CANCEL": "Cancel",
   "CHANGE": "Change",
+  "CHANGE_EMAIL": "Change email",
+  "CHANGE_PASSWORD": "Change password",
+  "CITY_DISTRICT": "City district",
   "CLEAR_FILTERS": "Clear Filters",
   "CLOSE_RECRUITMENT": "Closed Recruitment",
   "CONFIRM": "Confirm",
   "CONTACTS": "CONTACTS",
+  "DATE_OF_BIRTH_OF_THE_MANAGER": "Date of birth of the manager",
```
Comment (litvinets) [inline src/assets/i18n/en.json:27]: "DIRECTOR_BIRTHDAY": "Date of birth of the director",

**Review by litvinets — CHANGES_REQUESTED** [review]

**Comment by litvinets** [comment]
@khrMykhailo  I have fixed the comments above and merged the PR, so we can reuse all the amazing work you have done further for the next localization steps! Thank you!! ♥

**Review by litvinets — APPROVED** [review]

**Pull request merged**